SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -467,6 +467,34 @@ bool USpatialClassInfoManager::IsSublevelComponent(Worker_ComponentId ComponentI
 	return SchemaDatabase->LevelComponentIds.Contains(ComponentId);
 }
 
+const FClassInfo * USpatialClassInfoManager::GetClassInfoForNewSubobject(UObject * Object, Worker_EntityId EntityId, USpatialPackageMapClient* PackageMapClient)
```

Comment:
Can this function be made const? If so there are places where we pass around a pointer to the classInfoManager that can be const as well. I will mark them.

Reply:
Oh I imagine GetOrCreateClassInfoByClass modifies the manager.

Reply:
So it doesn't actually modify the manager, I think, but we would have to make a bunch of things const functions.